rdflib/graph.py

```diff
@@ -1762,7 +1762,7 @@ def add(self, triple):
 
     def addN(self, quads):
         """Add a sequence of triple with context"""
-
+        
```

Comment:
Why did you add a newline here?

Reply:
Fixed this. It wasn't required. I was just going through the code.